### ISSUE TITLE
ccls --version reports (apparently) only the LLVM version

### ISSUE THREAD

**Issue by madscientist** [MaskRay/ccls#334]
### Observed behavior

Running `ccls --version` shows just the LLVM version info:
```
$ ccls --version
ccls version 7.0.1 (tags/RELEASE_701/final)
```

### Expected behavior

Showing the LLVM version is useful, but I'd really like to see information on the version of ccls (e.g., the output of `git describe --tag`).  It's not always the case that I have ready access to the ccls Git repository that I built from when I need to know the version of ccls I'm using.

**Comment by madscientist** [comment]
I can create a PR for this over the weekend if that would be OK.

**Comment by MaskRay** [comment]
Thank you! That will be very useful!

**Comment by MaskRay** [comment]
Fixed by #342
